src/renderer/components/ft-share-button/ft-share-button.js

```diff
@@ -15,6 +15,15 @@ export default Vue.extend({
     'ft-toggle-switch': FtToggleSwitch
   },
   props: {
+    mediaType: {
```

Comment:
`shareTargetType`?
Channel doesn't seem to be a media type to me...

Reply:
You're right !  It's updated